specification/web/resource-manager/Microsoft.Web/stable/2023-01-01/ResourceProvider.json

```diff
@@ -507,6 +507,54 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/locations/{location}/quotaType/{quotaType}/usages": {
+      "get": {
```

Comment:
add x-ms-pageable for collection-get

https://github.com/Azure/azure-openapi-validator/blob/main/docs/pageable-operation.md